src/MediaWiki/PageInfoProvider.php

```diff
@@ -66,6 +66,10 @@ public function getModificationDate() {
 	 * @return integer
 	 */
 	public function getCreationDate() {
+		if ( $this->wikiPage->getTitle()->getFirstRevision() === null ) {
+			return wfTimestamp( TS_MW, 0 );
+		}
```

Comment:
This looks good as a local fix though I'm wondering if it is treating a symptom rather than the root cause. I do not know the code well enough to tell. One for @mwjames